kie-server-parent/kie-server-controller/kie-server-controller-mgmt-client/src/main/java/org/kie/server/controller/management/client/rest/RestKieServerMgmtControllerClient.java

```diff
@@ -68,83 +80,57 @@
     private Client httpClient;
     protected Marshaller marshaller;
 
-    public KieServerMgmtControllerClient(String controllerBaseUrl, String login, String password) {
+    public RestKieServerMgmtControllerClient(String controllerBaseUrl, String login, String password) {
         this(controllerBaseUrl, login, password, DEFAULT_MARSHALLING_FORMAT);
     }
 
-    public KieServerMgmtControllerClient(String controllerBaseUrl, String login, String password, MarshallingFormat format) {
+    public RestKieServerMgmtControllerClient(String controllerBaseUrl, String login, String password, MarshallingFormat format) {
         this.controllerBaseUrl = controllerBaseUrl;
-        httpClient = new ResteasyClientBuilder()
-                .establishConnectionTimeout(10, TimeUnit.SECONDS)
-                .socketTimeout(60, TimeUnit.SECONDS)
```

Comment:
what's the reason to remove the timeout settings? Isn't this no longer needed? It might be of an issue on some environments where particular request (like a first one) takes significantly more time...

Reply:
I suppose it is because JAX-RS 2.0 API doesn't have timeout definition support. It is brought in JAX-RS 2.1 : https://github.com/jax-rs/api/commit/d18207327468870ab885120c3790e36a741af26d

Reply:
ok, thanks for explaining @sutaakar

Reply:
@mswiderski for JAX-RS 2.0 you can specify timeout settings via properties, which are vendor specific, see: http://www.adam-bien.com/roller/abien/entry/setting_timeout_for_the_jax and https://www.ibm.com/support/knowledgecenter/en/SSEQTP_8.5.5/com.ibm.websphere.wlp.doc/ae/twlp_jaxrs2.0_clientconfig.html
I will add a method to allow that these properties are specified.